.github/workflows/mvn-verify-check.yml

```diff
@@ -38,18 +38,21 @@ env:
 jobs:
   cache-dependencies:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java-version: [8, 17]
```

Comment:
I suggest deal with github cache optimization in a separate PR. I think once you revert to JDK8-only cache-dependencies it will pass

Reply:
yeah, we need to differentiate the cache key of the old jdk8 cache from the new scala213+jdk17 runs.
this could be another change, feel free to file an issue for me, thanks!